instrumentation/couchbase/couchbase-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/couchbase/v2_0/CouchbaseAttributesGetter.java

```diff
@@ -48,4 +50,23 @@ public String getDbQueryText(CouchbaseRequestInfo couchbaseRequest) {
   public String getDbOperationName(CouchbaseRequestInfo couchbaseRequest) {
     return couchbaseRequest.operation();
   }
+
+  @Nullable
+  @Override
+  public String getResponseStatusFromException(Throwable throwable) {
+    if (throwable instanceof CouchbaseException) {
+      // todo
```

Comment:
@laurit can you provide a hint how I could get details of a newer library version?

Should this be in one of the other couchbase modules? Is there an example?